examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -39,6 +40,8 @@ constexpr chip::FabricId kCommissionerGammaFabricId = 3;
 
 CHIP_ERROR CHIPCommand::Run()
 {
+    chip::Controller::ExampleOperationalCredentialsIssuer opCredsIssuer;
```

Comment:
This is not the scope in which this thing is used...  It really does need to be member variable of CHIPCommand for the lifetime to be right, no?